WooCommerce/Classes/POS/Presentation/Card Present Payments/Reader Messages/PointOfSaleCardPresentPaymentDisconnectedMessageViewModel.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+struct PointOfSaleCardPresentPaymentDisconnectedMessageViewModel {
```

Comment:
```suggestion
struct PointOfSaleCardPresentPaymentReaderDisconnectedMessageViewModel {
```
I know it's a long name but better to be specific IMO.